Translation Editor/translation_lt.json

```diff
@@ -0,0 +1,198 @@
+				"nustatymai"
+			],
+			"desc": "Išplėstiniai nustatymai"
+		}
+	},
+	"menuOptions": {
+		"PowerSource": {
+			"text": "",
+			"text2": [
+				"Maitinimo",
+				"šaltinis"
+			],
+			"desc": "Išjungimo įtampa. <DC 10V arba celių (S) kiekis (3.3V per celę)>"
+		},
+		"SleepTemperature": {
+			"text": "",
+			"text2": [
+				"Miego",
+				"temperat."
+			],
+			"desc": "Miego temperatūra <C>"
+		},
+		"SleepTimeout": {
+			"text": "",
+			"text2": [
+				"Miego",
+				"laikas"
+			],
+			"desc": "Miego laikas <minutės/sekundės>"
+		},
+		"ShutdownTimeout": {
+			"text": "",
+			"text2": [
+				"Išjungimo",
+				"laikas"
+			],
+			"desc": "Išjungimo laikas <minutės>"
+		},
+		"MotionSensitivity": {
+			"text": "",
+			"text2": [
+				"Jautrumas",
+				"judesiui"
+			],
+			"desc": "Jautrumas judesiui <0 - išjungta, 1 - mažiausias, 9 - didžiausias>"
+		},
+		"TemperatureUnit": {
+			"text": "",
+			"text2": [
+				"Temperatūros",
+				"vienetai"
+			],
+			"desc": "Temperatūros vienetai <C - Celsijus, F - Farenheitas>"
+		},
+		"AdvancedIdle": {
+			"text": "",
+			"text2": [
+				"Detalus lauki-",
+				"mo ekranas"
+			],
+			"desc": "Ar rodyti papildomą informaciją mažesniu šriftu laukimo ekrane"
+		},
+		"DisplayRotation": {
+			"text": "",
+			"text2": [
+				"Ekrano",
+				"orientacija"
+			],
+			"desc": "Ekrano orientacija <A - automatinė, K - kairiarankiams, D - dešiniarankiams>"
+		},
+		"BoostEnabled": {
+			"text": "",
+			"text2": [
+				"Turbo režimas",
+				"įjungtas"
+			],
+			"desc": "Ar lituojant viršutinis mygtukas įjungia turbo režimą"
+		},
+		"BoostTemperature": {
+			"text": "",
+			"text2": [
+				"Turbo",
+				"temperat."
+			],
+			"desc": "Temperatūra turbo režimu"
+		},
+		"AutoStart": {
+			"text": "",
+			"text2": [
+				"Auto",
+				"paleidimas"
+			],
+			"desc": "Ar pradėti kaitininti iš karto įjungus lituoklį"
+		},
+		"CooldownBlink": {
+			"text": "",
+			"text2": [
+				"Atvėsimo",
+				"mirksėjimas"
+			],
+			"desc": "Ar mirksėti temperatūrą ekrane kol vėstantis antgalis vis dar karštas"
+		},
+		"TemperatureCalibration": {
+			"text": "",
+			"text2": [
+				"Kalibruoti",
+				"temperatūrą?"
+			],
+			"desc": "Antgalio temperatūros kalibravimas"
+		},
+		"SettingsReset": {
+			"text": "",
+			"text2": [
+				"Atstatyti",
+				"nustatymus?"
+			],
+			"desc": "Nustatyti nustatymus iš naujo"
+		},
+		"VoltageCalibration": {
+			"text": "",
+			"text2": [
+				"Kalibruoti",
+				"įvesties įtampą?"
+			],
+			"desc": "Įvesties įtampos kalibravimas. Trumpai paspauskite, norėdami nustatyti, ilgai paspauskite, kad išeitumėte"
+		},
+		"AdvancedSoldering": {
+			"text": "",
+			"text2": [
+				"Detalus lita-",
+				"vimo ekranas"
+			],
+			"desc": "Ar rodyti išsamią informaciją lituojant"
+		},
+		"ScrollingSpeed": {
+			"text": "",
+			"text2": [
+				"Greitas apra-",
+				"šymo slinkimas"
+			],
+			"desc": "Greitis, kuriuo šis tekstas slenka"
+		}
+	},
+	"languageLocalName": "Lietuviškai"
```

Comment:
Should be `"languageLocalName": "Lietuvių"`

Reply:
Thanks, @jvitkauskas! That was Google translate. ;) Fixed it. Probably there will be more errors like this one on a few other languages as well.

Reply:
Hi, you have missed a comma after "languageLocalName": "Lietuvių". JSON is not valid anymore.

Reply:
Yup, you are right again. Fixed. Sorry for that.